compose/bin/setup

```diff
@@ -44,16 +40,16 @@ if [ -z "$PUBLIC_KEY" ] || [ -z "$PRIVATE_KEY" ]; then
 fi
 
 if [ -n "$PUBLIC_KEY" ] && [ -n "$PRIVATE_KEY" ]; then
-    bin/clinotty composer config http-basic.repo.magento.com $PUBLIC_KEY $PRIVATE_KEY
+    bin/composer config http-basic.repo.magento.com $PUBLIC_KEY $PRIVATE_KEY
```

Comment:
Automated installs require the use of the `-T` flag when calling exec with docker compose, otherwise they will fail. I'd suggest to change the `bin/composer` script to use `bin/clinotty` instead of `bin/cli`, however it adds some nice pretty colors when running `bin/composer` that we'll lose if we switch. So, I think we should back out any changes in this `bin/setup` script that involved removing the `bin/clinotty` command.

Reply:
@markshust  Thanks for your suggestions